keyboards/jukaie/jk01/config.h

```diff
@@ -0,0 +1,115 @@
+/* Copyright 2023 JUKAIE <lice@jukaie.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define MATRIX_ROW_PINS \
+    { B0, B1, B2, B3, B4, B5, B6, B7, B8, B9, B10, B12 }
+#define MATRIX_COL_PINS \
+    { A0, A1, A2, A3, A4, A8, A9, A10 }
+
+#define ENCODERS_PAD_A \
+    { C2 }
+#define ENCODERS_PAD_B \
+    { C3 }
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Hold ESC on start up to clear EEPROM and boot into bootloader mode */
+#define BOOTMAGIC_LITE_ROW 1
+#define BOOTMAGIC_LITE_COLUMN 3
+
+#define TAP_CODE_DELAY 10
+
```

Comment:
This should all be migrated to `info.json`. You can use `qmk info -f json -kb jukaie/jk01` to help.
```suggestion
```